src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="product-card__imac"
+      >
+      <h2 class="product-card__description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <div class="product-card__code">Код товара: 195434</div>
+      <div class="product-card__rating">
+        <div class="stars">
```

Comment:
Лучше добавить модификатор количества активных звезд, в этом случае stars--4
```suggestion
        <div class="stars stars--4">
```